.travis.yml

```diff
@@ -53,6 +53,8 @@ before_install:
   - export RUSTC_WRAPPER=sccache
 
 jobs:
+  allow_failures:
+  -name: publish-with-rust-1.36
```

Comment:
Does it imply inability to use anchoring with Rust compiler 1.36.0?

Reply:
This means that while the git version of exonum is used, this test will always be failed.